engine/apps/grafana_plugin/helpers/client.py

```diff
@@ -69,30 +100,45 @@ def request_headers(self) -> dict:
 
 
 class GrafanaAPIClient(APIClient):
+    USER_PERMISSION_ENDPOINT = f"api/access-control/users/permissions?actionPrefix={ACTION_PREFIX}"
```

Comment:
@gamab do you mind just validating the latest URL to fetch the current user's permissions? 

I assume the response would still look something like this?
```json
[
  {
    "action": "some-perm"
  },
  {
    "action": "some-other-perm"
  }
]
```